com.avaloq.tools.ddk.xtext.ui/META-INF/MANIFEST.MF

```diff
@@ -1,31 +1,34 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: com.avaloq.tools.ddk.xtext.ui
-Bundle-SymbolicName: com.avaloq.tools.ddk.xtext.ui;singleton:=true
-Bundle-Version: 1.1.0.qualifier
-Bundle-Vendor: Avaloq Evolution AG
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-ActivationPolicy: lazy
-Require-Bundle: com.avaloq.tools.ddk.xtext,
- org.eclipse.xtext.ui,
- org.eclipse.xtext.ui.shared,
- org.eclipse.search,
- org.eclipse.ui.ide,
- org.apache.commons.lang,
- com.avaloq.tools.ddk.check.runtime.ui;visibility:=reexport
-Export-Package: com.avaloq.tools.ddk.xtext.ui,
- com.avaloq.tools.ddk.xtext.ui.editor,
- com.avaloq.tools.ddk.xtext.ui.editor.findrefs,
- com.avaloq.tools.ddk.xtext.ui.editor.model,
- com.avaloq.tools.ddk.xtext.ui.editor.model.edit,
- com.avaloq.tools.ddk.xtext.ui.editor.templates,
- com.avaloq.tools.ddk.xtext.ui.hover,
- com.avaloq.tools.ddk.xtext.ui.labeling,
- com.avaloq.tools.ddk.xtext.ui.outline,
- com.avaloq.tools.ddk.xtext.ui.quickfix,
- com.avaloq.tools.ddk.xtext.ui.syntaxcoloring,
- com.avaloq.tools.ddk.xtext.ui.templates,
- com.avaloq.tools.ddk.xtext.ui.util,
- com.avaloq.tools.ddk.xtext.ui.validation,
- com.avaloq.tools.ddk.xtext.ui.validation.preferences
-Import-Package: org.apache.log4j
+Manifest-Version: 1.0

```

Comment:
This looks very suspicious. Double-check the line endings.
@rmitin, do we want to conform these to a particular style or keep them unchanged?

Reply:
The line endings have changed from LF to CRLF. I've polled a few other files in DDK: most manifests used LF line endings but all source files used CRLF. So I don't know whether to change these back to LF or not. @rmitin ?